include/pisa/invert.hpp

```diff
@@ -403,19 +401,16 @@ void merge_batches(std::string const &output_basename, uint32_t batch_count, uin
     }
 }
 
-void invert_forward_index(std::string const &input_basename,
-                          std::string const &output_basename,
-                          uint32_t           term_count,
-                          size_t             batch_size,
-                          size_t             threads)
+void invert_forward_index(InvertSettings settings)
```

Comment:
Same as above. 

My idea is that whatever stays in `include/pisa` should not be CLI related...

Reply:
See the two above comments.